### PR TITLE
Pioneer gliders

### PR DESCRIPTION
**Pull request by cdobs**

### CODE REVIEW

File: CP05MOAS-GL339/CP05MOAS-GL339_R00015_ingest.csv [hunk 1]
```diff
@@ -0,0 +1,7 @@
+parser,filename_mask,reference_designator,data_source,status,notes
+mi.dataset.driver.moas.gl.engineering.glider_eng_glider_recovered_driver,/omc_data/whoi/OMC/CP05MOAS-GL339/R00015/merged/cp_*.mrg,CP05MOAS-GL339-00-ENG000000,recovered_host,Available,
+mi.dataset.driver.moas.gl.adcpa.adcpa_m_glider_recovered_driver,/omc_data/whoi/OMC/CP05MOAS-GL339/R00015/dvl/*.PD0,CP05MOAS-GL339-01-ADCPAM000,recovered_host,Available,
+mi.dataset.driver.moas.gl.flort_m.flort_m_glider_recovered_driver,/omc_data/whoi/OMC/CP05MOAS-GL339/R00015/merged/cp_*.mrg,CP05MOAS-GL339-02-FLORTM000,recovered_host,Available,
+mi.dataset.driver.moas.gl.ctdgv.ctdgv_m_glider_recovered_driver,/omc_data/whoi/OMC/CP05MOAS-GL339/R00015/merged/cp_*.mrg,CP05MOAS-GL339-03-CTDGVM000,recovered_host,Available,
+mi.dataset.driver.moas.gl.dosta.dosta_abcdjm_glider_recovered_driver,/omc_data/whoi/OMC/CP05MOAS-GL339/R00015/merged/cp_*.mrg,CP05MOAS-GL339-04-DOSTAM000,recovered_host,Available,
+mi.dataset.driver.moas.gl.parad.parad_m_glider_recovered_driver,/omc_data/whoi/OMC/CP05MOAS-GL339/R00015/merged/cp_*.mrg,CP05MOAS-GL339-05-PARADM000,recovered_host,Available,
```
Comment (rtravis-whoi) [inline CP05MOAS-GL339/CP05MOAS-GL339_R00015_ingest.csv:7]: looks good; data available

File: CP05MOAS-GL388/CP05MOAS-GL388_R00013_ingest.csv [hunk 2]
```diff
@@ -0,0 +1,7 @@
+parser,filename_mask,reference_designator,data_source,status,notes
+mi.dataset.driver.moas.gl.engineering.glider_eng_glider_recovered_driver,/omc_data/whoi/OMC/CP05MOAS-GL388/R00013/merged/cp_*.mrg,CP05MOAS-GL388-00-ENG000000,recovered_host,Available,
+mi.dataset.driver.moas.gl.adcpa.adcpa_m_glider_recovered_driver,/omc_data/whoi/OMC/CP05MOAS-GL388/R00013/dvl/*.PD0,CP05MOAS-GL388-01-ADCPAM000,recovered_host,Available,
+mi.dataset.driver.moas.gl.flort_m.flort_m_glider_recovered_driver,/omc_data/whoi/OMC/CP05MOAS-GL388/R00013/merged/cp_*.mrg,CP05MOAS-GL388-02-FLORTM000,recovered_host,Available,
+mi.dataset.driver.moas.gl.ctdgv.ctdgv_m_glider_recovered_driver,/omc_data/whoi/OMC/CP05MOAS-GL388/R00013/merged/cp_*.mrg,CP05MOAS-GL388-03-CTDGVM000,recovered_host,Available,
+mi.dataset.driver.moas.gl.dosta.dosta_abcdjm_glider_recovered_driver,/omc_data/whoi/OMC/CP05MOAS-GL388/R00013/merged/cp_*.mrg,CP05MOAS-GL388-04-DOSTAM000,recovered_host,Available,
+mi.dataset.driver.moas.gl.parad.parad_m_glider_recovered_driver,/omc_data/whoi/OMC/CP05MOAS-GL388/R00013/merged/cp_*.mrg,CP05MOAS-GL388-05-PARADM000,recovered_host,Available,
```
Comment (rtravis-whoi) [inline CP05MOAS-GL388/CP05MOAS-GL388_R00013_ingest.csv:7]: looks good; data available.

File: CP05MOAS-GL388/CP05MOAS-GL388_R00014_ingest.csv [hunk 3]
```diff
@@ -0,0 +1,7 @@
+parser,filename_mask,reference_designator,data_source,status,notes
+mi.dataset.driver.moas.gl.engineering.glider_eng_glider_recovered_driver,/omc_data/whoi/OMC/CP05MOAS-GL388/R00014/merged/cp_*.mrg,CP05MOAS-GL388-00-ENG000000,recovered_host,Available,
+mi.dataset.driver.moas.gl.adcpa.adcpa_m_glider_recovered_driver,/omc_data/whoi/OMC/CP05MOAS-GL388/R00014/dvl/*.PD0,CP05MOAS-GL388-01-ADCPAM000,recovered_host,Available,
+mi.dataset.driver.moas.gl.flort_m.flort_m_glider_recovered_driver,/omc_data/whoi/OMC/CP05MOAS-GL388/R00014/merged/cp_*.mrg,CP05MOAS-GL388-02-FLORTM000,recovered_host,Available,
+mi.dataset.driver.moas.gl.ctdgv.ctdgv_m_glider_recovered_driver,/omc_data/whoi/OMC/CP05MOAS-GL388/R00014/merged/cp_*.mrg,CP05MOAS-GL388-03-CTDGVM000,recovered_host,Available,
+mi.dataset.driver.moas.gl.dosta.dosta_abcdjm_glider_recovered_driver,/omc_data/whoi/OMC/CP05MOAS-GL388/R00014/merged/cp_*.mrg,CP05MOAS-GL388-04-DOSTAM000,recovered_host,Available,
+mi.dataset.driver.moas.gl.parad.parad_m_glider_recovered_driver,/omc_data/whoi/OMC/CP05MOAS-GL388/R00014/merged/cp_*.mrg,CP05MOAS-GL388-05-PARADM000,recovered_host,Available,
```
Comment (rtravis-whoi) [inline CP05MOAS-GL388/CP05MOAS-GL388_R00014_ingest.csv:7]: looks good; data available.

**Review by rtravis-whoi — APPROVED** [review]

File: CP05MOAS-GL339/CP05MOAS-GL339_R00015_ingest.csv (hunk 1 above)
Comment (johnmlund) [inline CP05MOAS-GL339/CP05MOAS-GL339_R00015_ingest.csv:7]: ok!

File: CP05MOAS-GL388/CP05MOAS-GL388_R00014_ingest.csv (hunk 3 above)
Comment (johnmlund) [inline CP05MOAS-GL388/CP05MOAS-GL388_R00014_ingest.csv:7]: ok

File: CP05MOAS-GL388/CP05MOAS-GL388_R00013_ingest.csv (hunk 2 above)
Comment (johnmlund) [inline CP05MOAS-GL388/CP05MOAS-GL388_R00013_ingest.csv:7]: ok

**Review by johnmlund — APPROVED** [review]

**Pull request merged**